enkit/outputs/commands.go

```diff
@@ -303,5 +303,24 @@ func NewShutdown(root *Root) *Shutdown {
 }
 
 func (c *Shutdown) Run(cmd *cobra.Command, args []string) error {
-	return fmt.Errorf("`enkit outputs Shutdown` is unimplemented")
+	bbOpts := bbexec.NewClientOptions(
+		c.root.Log,
+		0, /* tunnel port does not matter in this case */
+		c.root.OutputsRoot,
+	)
+	bbClient, err := bbexec.MaybeStartClient(bbOpts)
+	if err != nil {
+		return err
+	}
+	var errs []error
+	if err := bbClient.Shutdown(); err != nil {
+		errs = append(errs, fmt.Errorf("error maybe? killing the process of existing bb_clientd %v", err))
+	}
+	if err := os.RemoveAll(c.root.OutputsRoot); err != nil {
```

Comment:
Is there a method on bbOpts we can use/make public instead? Incidentally, I think there's already code that effectively deletes everything inside OutputsRoot, but leaves OutputsRoot intact (the user might be surprised if OutputsRoot is gone after a shutdown instead of just empty)